lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h

```diff
@@ -123,9 +123,15 @@ DECLARE_ENUM(TunnelEvent, TUNNEL_EVENTS)
 
 #define BASE_EVENT_MODEL(XX, ...) \
 XX(identifier, string, none, identifier, __VA_ARGS__) \
-XX(event_type, TunnelEvent, none, type, __VA_ARGS__)
+XX(event_type, TunnelEvent, none, type, __VA_ARGS__)  \
+XX(code, int, none, code, __VA_ARGS__)
```

Comment:
this is only for context event, right? move it to ztx event

Reply:
moved the code to context